content/tutorials/secure-kubernetes-app-secrets-with-esc/index.md

```diff
@@ -19,6 +19,9 @@ prereqs:
     - Install and configure [Kubectl CLI](https://kubernetes.io/docs/reference/kubectl/overview)
     - Install and configure [Minikube](https://minikube.sigs.k8s.io/docs/start)
 estimated_time: 15
+collections:
+    - using-esc
+collections_weight: 1
```

Comment:
These are both `1`. Did you want one to sort reliably before the other? (If so, suggest making that explicit.)

Reply:
Good call. Wasn't worried about sorting, but probably a good idea to be more explicit about that.